assignment-2/7s.md

```diff
@@ -1,2 +1,171 @@
+
+* *Limitations* .-
+When something is misaligned , what are the step to follow?
+
+## 2. Through the eyes of 7S model.
+
+* **2.1 Software industry:**
+
+### Spotify
+
+Spotify is a commercial music streaming, podcast and video service, founded in april 2006 by Daniel Ek and Martin Lorentzon. Spotify was launched in September 2008 and by June 2015 Spotify had more than 75 million active users, including about 20 million paid users. 
+
+##### Strategy
+
+> • Capabilities. Gathering, sharing and evaluating information about low-cost
+> infrastructures and related activities.
+> 
+> • Social Networks. Building open and active coalitions among experts—from any
+> organization or part of the world—who have background and knowledge in key areas,
+> and among the business, government, and civil society stakeholders who will have to live
+> with and sustain the solutions on the ground.
+> 
+> • Policy, doctrine and operational procedures. Converting high-level guidance into
+> effective field procedures.
+> 
+> • Legal and regulatory issues. Complying with all laws and regulations, while identifying
+> and working to change those that impede cooperation and unity of effort.
+> 
+> • Resources. Identifying needs for both the short and long terms.
+> • Training, exercises, and education. Maintaining readiness, capturing lessons learned, and
+> preparing future leaders.
+
+Spotify
+
+The vision of Spotify can best be decribed in the following manner: give people access to all the music they want all the time, in a completely legal manner. It is their mission to ensure a fair digital deal for artists.
+
+- Two music streaming tiers: Spotify Free ( a lot advertisements) and Spotify Premium ( paid, remove advertisements, improve audio quality, download available), this is called 'freemium pricing strategy'.
+- Spotify pays artists based on their 'market share' (the number of streams for their songs as a proportion of total songs streamed on the service). They distribute approximately 70% to rights-holders.
+- Other competitors in the market like Grooveshark have been shut down as part of legal settlements, because strategies that did not compensate the music production, Spotify even having some artist criticism, has manage to hold some balance between their activities and the music production industry.
+
+##### Structure
+
+> Structure is about how the organization is arranged or configured. Many organizations take a
+> functional structure, in which different branches of the organization specialize in particular
+> functions, such as manufacturing, sales, and service. Another common form is a divisional
+> structure, in which divisions are arrayed according to geographical regions, products, or some
+> other factor, and each division contains a full complement of functional elements (e.g., sales,
+> service, R&D).
+
+Spotify headquarters resides in London and its handles, research and development departments are in Stockholm.
```

Comment:
Okay, we know where the office of Spotify is. But where is the explanation of the real structure of Spotify. How do they breakdown the work, from an organizational perspective?